docs/articles/clustering/split-brain-resolver.md

```diff
@@ -262,10 +262,60 @@ akka.cluster.split-brain-resolver.lease-majority {
 }
 ```
 
-A `Lease` is a type of distributed lock implementation.
+A `Lease` is a type of distributed lock implementation to decide which nodes are allowed to survive. Only one SBR instance can acquire the lease to make the decision to remain up. The other side will not be able to acquire the lease and will therefore down itself.
```

Comment:
`Lease` can also be used for other things like Akka.Cluster.Sharding and ClusterSingleton hand-over too - so I would add " in on the context of SBRs, the leader who acquires the `Lease` gets to make downing decisions for the entire cluster"